assets/src/application/ui/forms/ee-form/RenderFields.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { RenderFieldsProps } from './types';
+import RenderField from './RenderField';
+
+const RenderFields: React.FC<RenderFieldsProps> = ({ fields, namespace }) => {
+	return (
+		<div>
+			{fields.map(({ name, ...fieldProps }, key) => {
+				const fieldName = namespace ? `${namespace}.${name}` : name;
```

Comment:
What is `namespace` in context of `RenderFields` ?
Maybe fields name or smth different ?

Reply:
It's explained in types:
```ts
/**
 * If true, each field inside the section
 * will be saved as `${section.name}.{field.name}`
 */
namespaceFields?: boolean;
```